app/javascript/admin/controllers/config_controller.js

```diff
@@ -93,6 +93,39 @@ export default class ConfigController extends Controller {
     }
   }
 
+  displaySnackbar(message) {
+    return document.dispatchEvent(
+      new CustomEvent('snackbar:add', {
+        detail: { message },
+      }),
+    );
+  }
+
+  async activateLightConfirmation(event) {
```

Comment:
nitpick (potentially-blocking): It's not clear to me what `activateLightConfirmation` means. Is this method called `activateLightConfirmation` because it's not a form post with a full server-side roundtrip? From what it looks like in the context of where this is called, `updateConfigurationSettings` or `updateSettings` seems more appropriate.

Reply:
I will change